src/features/cards/SortCards.ts

```diff
@@ -0,0 +1,22 @@
+import { ICardsContext } from '~/shared';
+
+export const sortCards = (cards: ICardsContext): ICardsContext => {
+  cards.sort((a, b) => {
```

Comment:
```suggestion
  return cards.slice().sort((a, b) => {
```
sort меняет массив, а не создает новый, и это часто становится причиной хитрых багов, поэтому принято делать так, чтобы сначала скоприровать массив, а потом его сортировать
в еслинте тоже есть правило, чтобы голый сорт не применялся вроде. У вас точно еслинт работает?